src/main/scala/com/gu/emailservices/DigitalPackEmailFields.scala

```diff
@@ -0,0 +1,81 @@
+package com.gu.emailservices
+
+import com.gu.i18n.Currency
+import com.gu.support.workers.model._
+import org.joda.time.DateTime
+
+// Output Json should look like this:
+//
+// {
+//  "To": {
+//    "Address": "gow5hckeqzc86qqx4rc@gu.com",
+//    "SubscriberKey": "gow5hckeqzc86qqx4rc@gu.com",
+//    "ContactAttributes": {
+//      "SubscriberAttributes": {
+//        "ZuoraSubscriberId": "A-S00045678",
+//        "Date of first payment": "3 August 2018",
+//        "Address 2": "",
+//        "Trial period": "14",
+//        "First Name": "first_GOW5hckeqZC86QQx4Rc",
+//        "Last Name": "last_GOW5hckeqZC86QQx4Rc",
+//        "Country": "United Kingdom",
+//        "SubscriberKey": "gow5hckeqzc86qqx4rc@gu.com",
+//        "Default payment method": "Credit/Debit Card",
+//        "Currency": "£",
+//        "Post Code": "N1 9AG",
+//        "Subscription term": "month",
+//        "City": "London",
+//        "Subscription details": "£11.99 every month",
+//        "EmailAddress": "gow5hckeqzc86qqx4rc@gu.com",
+//        "Payment amount": "11.99",
+//        "Address 1": "Kings Place"
+//      }
+//    }
+//  },
+//  "DataExtensionName": "digipack"
+//}
+
+case class DigitalPackEmailFields(
+    accountId: String,
+    billingPeriod: BillingPeriod,
+    user: User,
+    currency: Currency,
+    paymentMethod: PaymentMethod,
+    directDebitMandateId: Option[String] = None
+) extends EmailFields {
+
+  val amount = 11.99F //TODO: Get the real price
+  val subscriptionDetails = s"${currency.glyph}$amount every ${billingPeriod.noun}"
+  val paymentFields = paymentMethod match {
+    case dd: DirectDebitPaymentMethod => List(
+      "Account number" -> mask(dd.bankTransferAccountNumber),
```

Comment:
might be worth adding a helper function to build this list (looks like we do a similar thing twice)?

Reply:
Ah yes sorry, I didn't spot the slight distinctions